### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by spesnova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## SUPPORTED TAGS
+
+* [`2.7`](2.7/Dockerfile)
+
+## HOW TO USE
+Create a `Dockerfile` in your Ruby app project
+
+```
+FROM quay.io/wantedly/python:2.7
+CMD ["./your-daemon-or-script.rb"]
```
Comment (dtan4) [inline README.md]: Please replace `Ruby` and `.rb` to `Python` and `.py` .

Comment (spesnova) [inline README.md]: nice catch!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+For more information about this, please see [official one's README](https://github.com/docker-library/python).
+
+![](https://raw.githubusercontent.com/docker-library/docs/master/python/logo.png)
+
+## SUPPORTED TAGS
+
+* [`2.7`](2.7/Dockerfile)
+
+## HOW TO USE
+Create a `Dockerfile` in your Ruby app project
```
Comment (dtan4) [inline README.md]: And please fix this `Ruby` -> `Python` :bow:

Comment (spesnova) [inline README.md]: thx!!

**Comment by dtan4** [comment]
LGTM :+1:

**Pull request merged**